Xamarin.Forms.Core/Element.cs

```diff
@@ -107,10 +107,9 @@ public string StyleId
 			}
 		}
 
-		internal virtual ReadOnlyCollection<Element> LogicalChildren
-		{
-			get { return EmptyChildren; }
-		}
+		internal virtual ReadOnlyCollection<Element> LogicalChildrenInternal => EmptyChildren;
```

Comment:
Seems like that would be problematic for anything that needs to use `base`. It'd be doable, but kind of a pain for maintenance.